modules/s3db/msg.py

```diff
@@ -582,7 +582,8 @@ def model(self):
         tablename = "msg_email_channel"
         define_table(tablename,
                      # Instance
-                     super_link("channel_id", "msg_channel"),
+                     super_link("message_id", "msg_message"),
+                     self.msg_channel_id(),
```

Comment:
err, no.
This is NOT an instance of msg_message.
As an instance of msg_channel it doesn't need the self.msg_channel_id(), but rather should keep what ti is.
self.msg_channel_id is for components of channel